### PR TITLE
[Docs] Add instructions for automatic export of dashboard CSV

### PR DESCRIPTION
**Pull request by davea**
This wasn't covered in the docs already and has been asked a couple of times.

[skip changelog]

### CODE REVIEW

File: docs/_includes/admin-tasks-content.md [hunk 1]
```diff
@@ -863,6 +863,22 @@ Android device or via an iOS device).
...
 
+#### Importing dashboard data automatically
+
+You may wish to automatically import data from the dashboard into your system on a regular basis.
+Once you set your chosen criteria on the dashboard page, make a note of the URL of the "Reports"
+or "Updates" link in the "Export as CSV" section.
+Then you should generate an access token by visiting your account page and then "Security" and
+generating a token from there.
+
+You can then specify that token either by appending it the URL as an `access_token` parameter,
```
Comment (dracos) [inline docs/_includes/admin-tasks-content.md:874]: appending it *to* the URL

File: docs/_includes/admin-tasks-content.md [hunk 2]
```diff
@@ -863,6 +863,22 @@ Android device or via an iOS device).
...
+or including it as an `Authorization: Bearer <token>` HTTP header in the request from your system.
+
+As reports can take some time to generate, you can either specify `export=1` in your URL and it
```
Comment (dracos) [inline docs/_includes/admin-tasks-content.md:877]: Given you've given better instructions on where to get the starting URL from (which will already have export=2 in it), perhaps change this to "you can change the export parameter to 1" and then "leave it as export=2" later on, maybe?

**Review by dracos — CHANGES_REQUESTED** [review]
One typo, one suggestion, thanks for the idea

File: docs/_includes/admin-tasks-content.md (hunk 2 above)
Comment (davea) [reply docs/_includes/admin-tasks-content.md]: Thanks, I've reworded this section to be a bit clearer.

**Review by dracos — APPROVED** [review]

**Pull request merged**